src/lib/utils.js

```diff
@@ -63,3 +64,47 @@ export const getUrlFromString = (str) => {
   if (url.startsWith("<")) url = url.substring(1, url.length - 1);
   return url;
 };
+
+// returns the urls that are in the text
+export function getUrls(text) {
+  const matcher = /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()'@:%_\+.~#?!&//=]*)/gi;
```

Comment:
describe at high level what this regex does, if it conforms to a standard, does it support https vs http, etc